src/Microsoft.ML.Data/Model/ModelOperationsCatalog.cs

```diff
@@ -28,64 +28,69 @@ internal ModelOperationsCatalog(IHostEnvironment env)
             _env = env;
         }
 
+        private static TransformerChain<ITransformer> CreateChain(ITransformer transformer)
```

Comment:
>CreateChain [](start = 54, length = 11)

I can't find it, where are you are using this method?

Reply:
Ah thanks I'll nuke it